homeassistant/components/cloud/__init__.py

```diff
@@ -330,6 +332,7 @@ async def _on_initialized() -> None:
 
     account_link.async_setup(hass)
 
+    # Load legacy tts platform for backwards compatibility.
```

Comment:
How long do we need to maintain this compatibility? If it can be removed 2025.2 it makes sense to mention that also in this comment.

Reply:
We haven't decided if and when we want to deprecate it.